metron-platform/metron-solr/src/main/java/org/apache/metron/solr/dao/SolrUpdateDao.java

```diff
@@ -92,9 +93,17 @@ public void batchUpdate(Map<Document, Optional<String>> updates) throws IOExcept
     }
   }
 
-  private SolrInputDocument toSolrInputDocument(Document document) {
+  protected SolrInputDocument toSolrInputDocument(Document document) {
     SolrInputDocument solrInputDocument = new SolrInputDocument();
-    document.getDocument().forEach(solrInputDocument::addField);
+    document.getDocument().forEach((field, value) -> {
+      if (!isLocationSubField(field)) {
```

Comment:
Are we certain that location fields are the only type that have this issue?

Reply:
I don't see any other fields like this in the sensors we have but that doesn't mean there are not other field types like this.  I wasn't able to find any documentation around which fields Solr expands either.